docs/reference/verification-result-version.md

```diff
@@ -134,6 +134,9 @@ definitions:
       name:
         type: string
         description: The name of the verifier that performed the verification
+      type:
+        type: string
+        description: The type of the verifier that performed the verification
```

Comment:
we can add `optional` to type since it can be empty.
```suggestion
        description: The type of the verifier that performed the verification, optional
```